CLAUDE.md

```
@@ -52,16 +52,46 @@ Template variables use Handlebars syntax `{{variableName}}` and are expanded at
+|----------|---------------|-------------|
+| `Date` | `2026-02-04` | Current date (YYYY-MM-DD) |
+| `DateTime` | `2026-02-04T10:30:00.000Z` | Full ISO 8601 timestamp |
+| `Year` | `2026` | Current year |
+| `Month` | `02` | Current month (01-12) |
+| `Day` | `04` | Current day (01-31) |
+| `WorkPath` | `.work` | Default artifact directory |
+
+Built-in variables use PascalCase and can be overridden by any other source.
+
+**CLI Example:**
 ```bash
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)